openstack/clustering/v1/policies/requests.go

```diff
@@ -54,3 +54,46 @@ func List(client *gophercloud.ServiceClient, opts ListOptsBuilder) pagination.Pa
 		return p
 	})
 }
+
+// CreateOpts params
+type CreateOpts struct {
+	Name string `json:"name"`
+	Spec Spec   `json:"-"`
+}
+
+type Spec struct {
+	Description string
+	Properties  map[string]interface{}
+	Type        string
+	Version     string
+}
```

Comment:
2. Add `json` tags to each of these fields

Reply:
Done